addons/sourcemod/scripting/vsh/tags.sp

```diff
@@ -482,6 +482,7 @@ public void Tags_Climb(int iClient, int iTarget, TagsParams tParams)
 	float flHeight = tParams.GetFloat("height");
 	int iMax = tParams.GetInt("max");
 	float flDamage = tParams.GetFloat("selfdamage");
+	float flHorizontalSpeedMult = tParams.GetFloat("speed");
```

Comment:
"horizontal" may be a better name than "speed", otherwise it may be confused with "height".

Also default value 1.0 should be set here instead of requiring config to set
`tParams.GetFloat("horizontal", 1.0);`